.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Report Rancher Desktop issue
+title: "[Bug]- "
+labels: "bug :bug:"
```

Comment:
Can the label just be `bug` with now emoji. This would match the existing label we have.

Reply:
Should be possible. Let me try!